docs/style.css

```diff
@@ -0,0 +1,322 @@
+@font-face {
+  font-family: "Press Start 2P";
+  src: local("Press Start 2P"), url(https://fonts.googleapis.com/css?family=Press+Start+2P), cursive;
```

Comment:
I don't think this is working correctly in Chrome.
Print [here](https://ik.imagekit.io/6xhf1gnexgdgk/screencapture-localhost-4167-2019-02-25-12_29_11_H1VBqKWUE.png)

Reply:
😱 

I fixed `font-family`.

Reply:
Could you check again please? It still not working for me :thinking: 
**OS**: Ubuntu 18.04
**Chrome**: 72.0.3626.119 64bits

Reply:
Oh…sry, I was misunderstanding how to use Google fonts. 😅 

> Maybe add this?

The size of web fonts is large, so if we do that, `Performance` value will go down. because rendering is blocked by loading.

Reply:
Smart, I didn't thought of that. I will test again right away 👍